examples/get-started/pure-js/google-maps/app.js

```diff
@@ -12,44 +12,45 @@ const GOOGLE_MAPS_API_KEY = process.env.GoogleMapsAPIKey; // eslint-disable-line
 const GOOGLE_MAP_ID = process.env.GoogleMapsMapId; // eslint-disable-line
 
 const loader = new Loader({apiKey: GOOGLE_MAPS_API_KEY});
-const googlemaps = await loader.importLibrary('maps');
 
-const map = new googlemaps.Map(document.getElementById('map'), {
-  center: {lat: 51.47, lng: 0.45},
-  zoom: 5,
-  mapId: GOOGLE_MAP_ID
-});
+loader.importLibrary('maps').then(googlemaps => {
```

Comment:
Can you await this instead?

Reply:
This does work with vite, but eslint is unable to parse top-level awaits. Alternatively I can wrap the whole thing in an async function.

Reply:
Alright, this is fine